src/guides/v2.4/graphql/mutations/subscribe-email-to-newsletter.md

```diff
@@ -0,0 +1,79 @@
+---
+group: graphql
+title: subscribeEmailToNewsletter mutation
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+The `subscribeEmailToNewsletter` mutation allows subscription for guests and registered customers.
+
+## Syntax
+
+`mutation: {subscribeEmailToNewsletter(email: String!): SubscribeEmailToNewsletterOutput}`
+
+## Example usage
+
+The following call adds an email into a newsletter subscription.
+
+**Request:**
+
+```graphql
+mutation {
+  subscribeEmailToNewsletter(
+    email: "email@example.com"
+  ) {
+    status
+  }
+}
+```
+
+**Response:**
+
+```json
+{
+  "data": {
+    "subscribeEmailToNewsletter": {
+      "status": "SUBSCRIBED"
+    }
+  }
+}
+```
+
+## Input arguments
+
+Mutation contains a required `email` argument specified an email address which should be added into a newsletter subscription.
+
+## Output attributes
+
+The `SubscribeEmailToNewsletterOutput` object contains the following attributes:
+
+Attribute | Data Type | Description
+--- | --- | ---
+`status` | SubscriptionStatusesEnum | Contains a subscription status of specified `email` address.
+
+### SubscriptionStatusesEnum object
+
+In general, the `SubscriptionStatusesEnum` object can return one of the following values:
+
+Value | Description
+--- | ---
+`NOT_ACTIVE` | Subscription requires a confirmation. Confirmation email has been sent to specified email address to confirm the subscription.
+`SUBSCRIBED` | Email address is subscribed.
+`UNSUBSCRIBED` | Email address is unsubscribed.
+`UNCONFIRMED` | Specified email ties to customer which did not confirm a required customer registration.
+
+For the `subscribeEmailToNewsletter` mutation the `SubscriptionStatusesEnum` object may return only 2 of them:
```

Comment:
```suggestion
The `subscribeEmailToNewsletter` mutation for the `status` field may return only the following statuses:
```

Reply:
A minor adjustment just to mention that SubscriptionStatusesEnum cannot physically return anything since it's like an array.

Reply:
Yep.

**implemented**